python/plugins/processing/gui/GetScriptsAndModels.py

```diff
@@ -268,7 +268,13 @@ def setHelp(self, reply, item):
             html = self.tr('<h2>No detailed description available for this script</h2>')
         else:
             content = bytes(reply.readAll()).decode('utf8')
-            descriptions = json.loads(content)
+            try:
+                descriptions = json.loads(content)
+            except json.decoder.JSONDecodeError:
+                html = self.tr('<h2>JSON Decoding Error - could not load help</h2>')
+            except Exception:
+                html = self.tr('<h2>Unspecified Error - could not load help</h2>')
```

Comment:
I guess you didn't introduce this catch-all block here because you ran into it but more as a "let's be on the safe side" thing?
Since it's not really expected to happen (and if it does, we should provide means to debug and deal with it) should this provide some hints like the following?

```
except Exception as e:
    self.tr('<h2>Unspecified Error - could not load help</h2><p>{details}</p>').format(repr(e))
```